src/extension/src/azure/azure-app-service/appServiceProvider.ts

```diff
@@ -75,4 +91,68 @@ export class AppServiceProvider {
         return error.message;
       });
   }
+
+  // Creates a Basic Tier App Service Plan (ASP)
+  public async createAppServicePlan(
+    aspSelection: AppServicePlanSelection
+  ): Promise<string> {
+    this.setWebClient(aspSelection.subscriptionItem);
+    if (this.webClient === undefined) {
+      throw new AuthorizationError(CONSTANTS.ERRORS.WEBSITE_CLIENT_NOT_DEFINED);
+    }
+    const appServicePlanSelection: AppServicePlan = {
+      kind: OS.Linux,
```

Comment:
Maybe it maybe good if you could put a comment here to indicate we choose the default values or those are default values for future maintenance. You don't need to take this advice LOL